### ISSUE TITLE
PHP Fatal error:  Cannot override final method ParaTest\Runners\PHPUnit\BaseRunner::run()

### ISSUE THREAD

**Issue by furatamasensei** [tonysm/laravel-paratest#4]
Trying to install this package and followed the instruction the exact same way explained on README file, I got this error when trying to `composer dump -o`.

```
$ composer dump -o
Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
PHP Fatal error:  Cannot override final method ParaTest\Runners\PHPUnit\BaseRunner::run() in C:\Users\Me\Documents\Laravel\myapp\vendor\tonysm\laravel-paratest\src\Testing\Paratest\LaravelRunner.php on line 28

Fatal error: Cannot override final method ParaTest\Runners\PHPUnit\BaseRunner::run() in C:\Users\Me\Documents\Laravel\myapp\vendor\tonysm\laravel-paratest\src\Testing\Paratest\LaravelRunner.php on line 28
```

**Steps To Reproduce**
1. Clone [this repository](https://github.com/pratamatama/laravel-paratest-reproduce)
2. Follow the instructions given on the README

**Screenshot**
![image](https://user-images.githubusercontent.com/42239243/101212135-cf963500-36aa-11eb-90e2-51b94987b764.png)
![image](https://user-images.githubusercontent.com/42239243/101212183-e2106e80-36aa-11eb-822c-e7236fa48c1c.png)

**Specifications**
OS: Windows 10 Pro 64-bit (10.0, Build 19042)
PHP Version: "7.4.x"
Laravel Version: "8.12"
Laravel Paratest Version: "^0.0.1"
Fresh Project: No

**Additional Note**
This also occurs with fresh laravel project installation.

**Comment by tonysm** [comment]
Ah, yes, that runner needs to be updated. I thought I had pushed the fix to that, but guess not. Will fix it today. Sorry for the trouble. That file isn't necessary to run the tests in parallel, though. The custom runner just deletes the databases after the test runs.

**Comment by tonysm** [comment]
Hi there. This should be fixed at 0.0.2.
